apps/etl-func/src/domain/interfaces/tokenizer.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from "zod";
+
+export interface TokenizerClient {
+  tokenize(pii: string): Promise<string>;
+}
```

Comment:
maybe we can use more generic terms here

what about `maskSensitiveInfo`? in the future we can use these function also for masking sensitive info from `subject`